tools/setup/setup.sh

```diff
@@ -129,7 +129,6 @@ if [[ $DEVICE == "tpu" ]]; then
             python3 -m uv pip install -U crcmod
             # Copy libtpu.so from GCS path
             gsutil cp "$LIBTPU_GCS_PATH" "$libtpu_path"
-            exit 0
```

Comment:
Why is this removed?

Reply:
Won't the rest of the script override the custom libtpu here though? Since this is handling `MODE=libtpu-only`

Reply:
I can add an `unset LIBTPU_GCS_PATH` here instead of the removed `exit 0`